src/Routes/seasons.ts

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * Seasons controller
+ */
+//import * as assert from "assert";
+import * as express from "express";
+import { SqlDataProvider } from "../DataProviders/SqlDataProvider/SqlDataProvider";
+import { Configuration } from "../Common/Configuration";
+import { Networking } from "../Common/Helpers";
+import { IDataProvider } from "../DataProviders/IDataProvider";
+const router = express.Router();
+
+// TODO: Add a factory pattern
+const dataProvider: IDataProvider = new SqlDataProvider(Configuration.sqlConfig);
+
+router.get('/', (req: express.Request, res: express.Response) => {
+    dataProvider.getSeasonList()
+        .then((result) => {
+            Networking.setResponseHeaders(res);
```

Comment:
you usually don't need to set the content type/status code and stringify the JSON yourself if you're just returning simple JSON. You can replace lines 18-19 with a simple `res.send(result);`. See [the docs](https://expressjs.com/en/api.html#res.send) for more.

Reply:
That would work, but the response won't have the content type header.  I'd rather keep it as that for other clients in case they depend on this header being set.